dataset/src/main/scala/frameless/functions/AggregateFunctions.scala

```diff
@@ -7,12 +7,15 @@ import org.apache.spark.sql.{functions => untyped}
 
 trait AggregateFunctions {
 
+  private def typedColumnToAggregate[A: TypedEncoder, T](a: TypedColumn[T, A]): TypedAggregate[T, A] =
+    new TypedAggregate[T,A](a.expr)
+
   /** Creates a [[frameless.TypedColumn]] of literal value. If A is to be encoded using an Injection make
     * sure the injection instance is in scope.
     *
     * apache/spark
     */
```

Comment:
Yea, that's the rhs of `lit()`. I am open to making this more elegant ... I did try to have the same name and make them both work, but I wasn't able to get there. That's when I resorted to changing the name and calling it litAggr.